site/content/docs/5.0/examples/cover/cover.rtl.css

```diff
@@ -0,0 +1,88 @@
+/*
+ * Globals
+ */
+
+/* temp, until https://www.fontface.me/ combines these fonts into one family. */
+@font-face {
+  font-family: Dubai;
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.woff") format("woff");
+}
+@font-face {
+  font-family: Dubai;
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai/dubai.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai/dubai.woff") format("woff");
+}
+
+
+/* Custom default button */
+.btn-secondary,
+.btn-secondary:hover,
+.btn-secondary:focus {
+  color: #333;
+  text-shadow: none; /* Prevent inheritance from `body` */
+}
+
+
+/*
+ * Base structure
+ */
+
+body {
+  text-shadow: 0 .05rem .1rem rgba(0, 0, 0, .5);
+  box-shadow: inset 0 0 5rem rgba(0, 0, 0, .5);
+}
+
+.cover-container {
+  max-width: 42em;
+}
+
+
+/*
+ * Header
+ */
+
+.nav-masthead .nav-link {
+  padding: .25rem 0;
+  font-weight: 700;
+  color: rgba(255, 255, 255, .5);
+  letter-spacing: .07em;
```

Comment:
I thought `letter-spacing` was usually a bad idea for Arabic content: is this required by the custom font you used?

Reply:
https://twitter.com/Aqeel_AT/status/1363501583774978049

The issue is some fonts overlaying the individual glyphs. Therefore, when you make a text transparent, the overlayed areas become darker. This should not be an issue with the other examples. This one is a special case.